conda_build/environ.py

```diff
@@ -345,7 +345,11 @@ def system_vars(env_dict, prefix):
     if 'MAKEFLAGS' in os.environ:
         d['MAKEFLAGS'] = os.environ['MAKEFLAGS']
 
-    d['CPU_COUNT'] = get_cpu_count()
+    if 'CPU_COUNT' in os.environ:
+        d['CPU_COUNT'] = os.environ['CPU_COUNT']
```

Comment:
This is a good way to accomplish this.  Correct me if I'm wrong, but it means that recipes need to add stuff like:

make -j${CPU_COUNT}

right?  So we are now saying that people make the choice to be parallel explicitly?  I agree with that, but I just want to make sure that I understand it clearly.